packages/grid/src/vaadin-grid-sort-mixin.d.ts

```diff
@@ -13,4 +13,20 @@ export declare class SortMixinClass {
    * @attr {boolean} multi-sort
    */
   multiSort: boolean;
+
+  /**
+   * Use this property to determine the order in which the grid rows are sorted.
+   * It's visually indicated by numbers in grid sorters placed in column headers.
```

Comment:
```suggestion
    * Controls how columns are added to the sort order when using multi-sort.
    * The sort order is visually indicated by numbers in grid sorters placed in column headers.
```
I feel we could make this more clearer that it's about multi-sort, and specifically how columns are added to the sort order.